### DISCUSSION TITLE
Help converting nginx config to nginx proxy manager

### DISCUSSION THREAD

**Discussion by nxarmada** [NginxProxyManager/nginx-proxy-manager#1821] [Q&A]
I have this config and I am trying to get it to work with nginx proxy manager.  UNMS uses wss.  I have the UI for UNMS loading and working correctly I believe I am having issues with the proxy_headers. https://help.ui.com/hc/en-us/articles/115015690207-UISP-Reverse-Proxy

`map $http_upgrade $connection_upgrade {
  default upgrade;
  ''      close;
}

server {
  listen 80;
  server_name uisp.example.com;

  client_max_body_size 4G;

  location / {
    proxy_redirect off;
    proxy_set_header Host $host;
    proxy_pass http://127.0.0.1:8080/;
  }
}

server {
  listen 443 ssl http2;
  server_name uisp.example.com;

  ssl_certificate     /etc/letsencrypt/live/uisp.example.com/fullchain.pem;
  ssl_certificate_key /etc/letsencrypt/live/uisp.example.com/privkey.pem;

  ssl on;

  set $upstream 127.0.0.1:8443;

  location / {
    proxy_pass     https://$upstream;
    proxy_redirect https://$upstream https://$server_name;

    proxy_cache off;
    proxy_store off;
    proxy_buffering off;
    proxy_http_version 1.1;
    proxy_read_timeout 36000s;

    proxy_set_header Host $http_host;
    proxy_set_header Upgrade $http_upgrade;
    proxy_set_header Connection $connection_upgrade;
    proxy_set_header X-Real-IP $remote_addr;
    proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
    proxy_set_header X-Forwarded-Proto $scheme;
    proxy_set_header Referer "";

    client_max_body_size 0;
  }
}`

**Comment by sisaacrussell** [comment]
I had to abandon using this project because I couldn't resolve this same issue. If it's ever resolved, I'd love to return.
